src/SWP/SuperdeskBridgeBundle/Resources/config/services.yml

```diff
@@ -0,0 +1,9 @@
+services:
+
+    swp_sd_bridge.bridge:
+        class: SWP\SuperdeskBridgeBundle\Bridge\Bridge
+        arguments: [@swp_sd_bridge.client, %superdeskbridge%]
+
+    swp_sd_bridge.client:
+        class: SWP\SuperdeskBridgeBundle\Client\Client
+        arguments: [%superdeskbridge%]
```

Comment:
:+1: yeah, that would also remove the need for a config.yml. All required defaults are stored in the class itself anyway.